cheatsheets/Docker_Security_Cheat_Sheet.md

```diff
@@ -109,11 +109,11 @@ spec:
 
 As a Kubernetes cluster administrator, you can configure it using [Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/).
 
-### RULE \#4 - Add –no-new-privileges flag
+### RULE \#4 - Prevent in container privilege escalation
```

Comment:
```suggestion
### RULE \#4 - Prevent in-container privilege escalation
```
I'm no grammar expert but I think it's easier to read with a hyphen. It makes it clear that "in container" is one entity.